### PR TITLE
LPS-80660 fix argument

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

**Comment by gamerson** [comment]
ci:test:sf

**Comment by gamerson** [comment]
ci:test:relevant

**Comment by gamerson** [comment]
ci:reopen

**Comment by Ithildir** [comment]
ci:reopen

**Comment by gamerson** [comment]
ci:test:relevant

**Comment by gamerson** [comment]
ci:stop

**Comment by gamerson** [comment]
ci:test:sf

**Comment by gamerson** [comment]
ci:test:relevant

**Comment by gamerson** [comment]
ci:test:sf

**Comment by gamerson** [comment]
ci:test:relevant

File: modules/sdk/project-templates/project-templates/build.gradle [hunk 1]
```diff
@@ -160,6 +160,7 @@ test {
 	systemProperty "nodejs.npm.ci.registry", project.property("nodejs.npm.ci.registry")
 	systemProperty "nodejs.npm.ci.sass.binary.site", project.property("nodejs.npm.ci.sass.binary.site")
 	systemProperty "repository.url", System.properties["repository.url"]
+	systemProperty "rootProjectDir", rootProject.projectDir
```
Comment (Ithildir) [inline modules/sdk/project-templates/project-templates/build.gradle]: Wouldn't it be better to just pass a `repository.local.dir` system property instead?

Comment (gamerson) [reply modules/sdk/project-templates/project-templates/build.gradle]: But how do I get that passed into system properties?

Comment (gamerson) [reply modules/sdk/project-templates/project-templates/build.gradle]: I guess i can modify init.gradle?

Comment (Ithildir) [reply modules/sdk/project-templates/project-templates/build.gradle]: Simply copy how it's done in `build-buildscript.gradle`, and add the value as a system property in the `test` task.

**Comment by gamerson** [comment]
ci:test:sf

**Comment by gamerson** [comment]
ci:test:relevant

**Comment by gamerson** [comment]
ci:reopen

**Comment by gamerson** [comment]
ci:test:relevant

**Pull request closed without merging**